### DISCUSSION TITLE
Welcome to R_at_Grattan Discussions!

### DISCUSSION THREAD

**Discussion by wfmackey** [grattan/R_at_Grattan#57] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wfmackey** [comment]
@MattCowgill what the hell is this

**Comment by BaccaraEve** [comment]
Hi, I'm Baccara and I need help with R for a project I'm doing. I don't have a lot of experience and I cannot figure out which code to use with what. I understand how to do some things but putting code together to grab stuff from my data set is frustrating.

**Reply by wfmackey** [reply]
Hi @Baccara8 ! I think a really good place to start is the R for Data Science book, which you can find for free here: https://r4ds.had.co.nz/. Chapter 11 covers importing data!
